### ISSUE TITLE
Biber ignores xsv fields

### ISSUE THREAD

**Issue by moewew** [plk/biber#250]
Consider the following two MWEs
```
\documentclass{article}

\usepackage{filecontents}
\begin{filecontents*}{groups.dbx}
\DeclareDatamodelFields[type=field, format=xsv, datatype=keyword]{groups}
\DeclareDatamodelEntryfields{groups}
\end{filecontents*}
\usepackage[style=authoryear,backend=biber,datamodel=groups]{biblatex}

\begin{filecontents}{\jobname.bib}
@misc{A01,
  author = {Author, A.},
  year   = {2001},
  title  = {Alpha},
  groups = {Group1,Group2},
}
@misc{B02,
  author = {Buthor, B.},
  year   = {2002},
  title  = {Bravo},
  groups = {Group1},
}
@misc{C03,
  author = {Cuthor, C.},
  year   = {2003},
  title  = {Charlie},
  groups = {Group2},
}
\end{filecontents}

\addbibresource{\jobname.bib}


\begin{document}
\nocite{*}

\printbibliography
\end{document}
```
compiles fine when run with LaTeX, Biber, LaTeX, LaTeX, but contrary to what I expected the `.bbl` file does not contain any mentions of `groups` and their field values.

On the other hand
```
\documentclass{article}

\usepackage{filecontents}
\begin{filecontents*}{groups.dbx}
\DeclareDatamodelFields[type=field, format=xsv, datatype=literal]{groups}
\DeclareDatamodelEntryfields{groups}
\end{filecontents*}
\usepackage[style=authoryear,backend=biber,datamodel=groups]{biblatex}

\begin{filecontents}{\jobname.bib}
@misc{A01,
  author = {Author, A.},
  year   = {2001},
  title  = {Alpha},
  groups = {Group1,Group2},
}
@misc{B02,
  author = {Buthor, B.},
  year   = {2002},
  title  = {Bravo},
  groups = {Group1},
}
@misc{C03,
  author = {Cuthor, C.},
  year   = {2003},
  title  = {Charlie},
  groups = {Group2},
}
\end{filecontents}

\addbibresource{\jobname.bib}


\begin{document}
\nocite{*}

\printbibliography
\end{document}
```
lets Biber die and produces an empty `.bbl` file. A `biber --trace` did not yield anything out of the ordinary in the `.blg` file, but I get
```
Can't use an undefined value as a subroutine reference at C:\Users\Moritz\AppData\Local\Temp\par-4d6f7269747a\cache-b499addb80b3706a003bc9fcde51a97832c83aee\inc\lib/Biber/Input/file/bibtex.pm line 866.
```
stdout (though that is not captured when I try to redirect it to a file with `>` on Windows? I noticed that behaviour for a few other messages as well, could something be done about that?)

**Comment by plk** [comment]
Hmm, looking into it. The final comment about stdout is expected as what you are seeing is stderr so just do `biber foo > out.txt 2>&1`.

**Comment by plk** [comment]
You found two edge cases - nobody had tried a `field/xsv/literal` before and this was missing in the handler lookup. The other problem was caused by `keywords` being an awkward exception which for historical reasons has a different format in the `.bbl` to every other field. Both fixed in biber 2.13 in DEV (no need to update biblatex).

**Comment by moewew** [comment]
The MWEs compile fine now, thanks.
